site/en/docs/extensions/migrating/rhc/index.md

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
One other nit, can you rename the folder to "remote-hosted-code". Also would like to hear @jpmedley's opinion on whether this belongs into `migrate` or `concepts` in the new IA.